src/core/pressure_inline.hpp

```diff
@@ -296,14 +296,34 @@ inline void add_bonded_virials(Particle *p1)
       return;
     }
 
-    get_mi_vector(dx, p1->r.p, p2->r.p);
+    double a[3] = {p1->r.p[0], p1->r.p[1], p1->r.p[2]};
+    double b[3] = {p2->r.p[0], p2->r.p[1], p2->r.p[2]};
+    
+    //printf("Coordinates part 1: %f, %f, %f \n", a[0], a[1], a[2]);
+    //printf("Coordinates part 2: %f, %f, %f \n", b[0], b[1], b[2]);
+
+    double n_le_shifts = dround((a[1]-b[1]) * box_l_i[1]);
```

Comment:
This should go into the ifdefed block I guesse.

Reply:
done